identity/client/src/pages/authorizations/modals/OwnerSelection/index.tsx

```diff
@@ -10,7 +10,7 @@ import { FC } from "react";
 import { OwnerType } from "src/utility/api/authorizations";
 import { searchUser } from "src/utility/api/users";
 import { searchGroups } from "src/utility/api/groups";
-import { searchMapping } from "src/utility/api/mappings";
+import { searchMappingRule } from "src/utility/api/mappings";
```

Comment:
🔧 We should also change the `mappings` class here too to be `mappingRules`

Reply:
We are going to have a specific PR for UI, these changes was the minimum required to have API interactions still working.